src/main/resources/db/migration/V3__.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE task
+    ADD team_id INT NULL;
+
+ALTER TABLE user
+    ADD team_id INT NULL;
+
+ALTER TABLE task
+    ADD CONSTRAINT FK_TASK_ON_TEAM FOREIGN KEY (team_id) REFERENCES team (id);
+
+ALTER TABLE user
+    ADD CONSTRAINT FK_USER_ON_TEAM FOREIGN KEY (team_id) REFERENCES team (id);
```

Comment:
Missing lineending at last row. 
Do we need multiple migration files for one pull request or can we merge them into one version?

Reply:
I will merge them